src/System.IO.Compression/src/System/IO/Compression/ZipArchive.cs

```diff
@@ -392,13 +372,8 @@ private void AddEntry(ZipArchiveEntry entry)
             }
         }
 
-
-        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1811:AvoidUncalledPrivateCode", Justification = "This code is used in a contract check.")]
-        internal bool IsStillArchiveStreamOwner(ZipArchiveEntry entry)
-        {
-            return _archiveStreamOwner == entry;
-        }
-
+        [SuppressMessage("Microsoft.Performance", "CA1811:AvoidUncalledPrivateCode", Justification = "This code is used in a contract check.")]
+        internal bool IsStillArchiveStreamOwner(ZipArchiveEntry entry) => _archiveStreamOwner == entry;
```

Comment:
Instead of a SuppressMessage, maybe this function and the Debug.Assert that accesses it should be wrapped in ```#if DEBUG```?